docs/index.rst

```diff
@@ -189,14 +189,16 @@ API Documentation
 
   This class handles HTTP Basic authentication for Flask routes.
 
-  .. method:: __init__(scheme=None, realm=None)
+  .. method:: __init__(scheme=None, realm=None, header=None)
```

Comment:
This I see as a problem. The `header` argument should only be exposed by the Token method.

Reply:
sorry missed this one, it supposed to be under `token`. I've fixed it.